app/controllers/api/stories_controller.rb

```diff
@@ -17,6 +17,22 @@ def after_create_resource(res)
     res.create_story_distributions
   end
 
+  def after_update_resource(res)
+    dist_tmpl = res
+      .try(:series)
+      .try(:distributions)
+      .try(:find) { |d| d.kind == "podcast" }
```

Comment:
could there be more than one? I mean, I think the models support it, even if we don't do that on the UI.
I think this works, but I wonder if it would be good to get a list in dist_tmpl of ids

Reply:
as I think about it, I think this logic of catching up on missing distributions is not specific to podcasts - we just don't have other distros at the moment, so I wonder if filtering on "podcasts" is necessary - maybe we should do this for all series distributions?